docs/cli/build.md

```diff
@@ -133,6 +133,58 @@ RUN --mount=type=secret,id=netrc,mode=0666,dst=/home/app/.netrc \
 
 Bear in mind that at this time, `GITHUB_TOKEN` in a GitHub Action cannot be used to clone other repositories, even within the same organisation.
 
+### Private npm modules
+
+Get the OpenFaaS Pro plugin and enable it:
+
+```bash
+faas-cli plugin get pro
+faas-cli pro enable
+```
+
+Create a function:
+
+```bash
+export OPENFAAS_PREFIX=openfaasltd
+faas-cli template store pull node17
+faas-cli new --lang python3-http withprivatenpm
+mv withprivatenpm.yml stack.yml
+```
+
+Edit your `~/.npmrc` file to include the authentication token for your npm registry.
+
+```
+//registry.npmjs.org/:_authToken=TOKEN
+```
+
+Add the `.npmrc` file as a build secret to the `stack.yml` file:
+
+```yaml
+version: 1.0
+provider:
+  name: openfaas
+  gateway: http://127.0.0.1:8080
+functions:
+  withprivatenpm:
+    lang: node17
+    handler: ./withprivatenpm
+    image: openfaasltd/withprivatenpm:latest
+    build_secrets:
+      npmrc: /home/welteki/.npmrc
```

Comment:
Can you use the string interpolation we talked about on the last community call, and update the previous Python example too?